src/components/Dropdown/dropdown.module.css

```diff
@@ -0,0 +1,69 @@
+.dropdown_wrapper {
```

Comment:
Can you create the variable for colors?

Reply:
this wrapper is not using any color where you are trying point it out please tell me

Reply:
L12, L22

Reply:
those colors are only being used in this component only so why we need to add it in the variables and variables are used if the same color are being used in multiple times